### PR TITLE
Combinato container

### PR DESCRIPTION
**Pull request by vncntprvst**
Runs and passes [installation test](https://github.com/jniediek/combinato/wiki/Installation-on-Linux#4-testing-the-installation)
Will probably need some tweaks to run with spikeinterface

### CODE REVIEW

File: combinato/build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker build -t spikeinterface/combinato:latest -t spikeinterface/combinato:0.1.0 .
```
Comment (alejoe91) [inline combinato/build.sh:3]:
```suggestion
docker build -t spikeinterface/combinato-base:latest -t spikeinterface/combinato-base:0.1.0 .
```

File: combinato/push.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker push --all-tags spikeinterface/combinato
```
Comment (alejoe91) [inline combinato/push.sh:3]:
```suggestion
docker push --all-tags spikeinterface/combinato-base
```

**Comment by bendichter** [comment]
Please add test here

**Comment by vncntprvst** [comment]
Will do

**Comment by vncntprvst** [comment]
From today's test, I get this error:

**Comment by alejoe91** [comment]
Yep, can you trtmy to generate the toy data with `num_channels=1`?

**Comment by vncntprvst** [comment]
It seems to work with `num_channels=1` :)

**Comment by vncntprvst** [comment]
@alejoe91 The container works for `num_channels=1`  
image

**Pull request merged**